crates/admin/src/cluster_controller/service.rs

```diff
@@ -417,10 +443,15 @@ async fn signal_all_partitions_started(
 
 #[cfg(test)]
 mod tests {
-    use super::Service;
+    use std::sync::atomic::{AtomicU64, Ordering};
```

Comment:
Oops, this was unintentional - I will back this out in the next iteration. I have configured my rustfmt to follow the suggested imports order and I'm fairly happy with the results; now it mostly only affects places where we didn't follow the conventions ourselves.